### DISCUSSION TITLE
Be able to configure a sound to play when a player gets a /msg

### DISCUSSION THREAD

**Discussion by ZepsiZola** [EssentialsX/Essentials#4790] [Ideas and Feature Suggestions]
I feel like we should be able to configure a sound for when a player gets a /msg.
Like it would be super useful if you have the game running in the background and you get a 'ding' and realize you have been messaged in the game.

The config in config.yml could look something like
`msg_sound: ''` for no sound and `msg_sound: 'SOUND'` to configure a specific sound.

**Comment by osfanbuff63** [comment]
In VentureChat's msg system, this happens, and I've always wanted it to be a part of EssentialsX's msg system, although for sure optionally.
